app/controllers/data_lists_controller.rb

```diff
@@ -15,7 +15,7 @@ def index
   # GET /data_lists/1
   # GET /data_lists/1.xml
   def show
-    @data_list = DataList.find_by id: params[:id]
+    @data_list = DataList.joins("inner join users on users.id = data_lists.owned_by").select("data_lists.*, users.email").find_by(id: params[:id])
```

Comment:
Actually, this join excludes any data_lists that does not have an owner assigned. I don't think that's what we want either. I vote for reverting this line.